[2025-06-27T07:20:37.553+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-27T07:20:37.650+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_processing_pipeline.Transform_Data_With_Spark manual__2025-06-27T07:14:33.133832+00:00 [queued]>
[2025-06-27T07:20:37.665+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_processing_pipeline.Transform_Data_With_Spark manual__2025-06-27T07:14:33.133832+00:00 [queued]>
[2025-06-27T07:20:37.666+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-06-27T07:20:37.695+0000] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): Transform_Data_With_Spark> on 2025-06-27 07:14:33.133832+00:00
[2025-06-27T07:20:37.703+0000] {standard_task_runner.py:64} INFO - Started process 670 to run task
[2025-06-27T07:20:37.709+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'spark_processing_pipeline', 'Transform_Data_With_Spark', 'manual__2025-06-27T07:14:33.133832+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpeolgylwm']
[2025-06-27T07:20:37.715+0000] {standard_task_runner.py:91} INFO - Job 17: Subtask Transform_Data_With_Spark
[2025-06-27T07:20:37.799+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-27T07:20:37.919+0000] {task_command.py:426} INFO - Running <TaskInstance: spark_processing_pipeline.Transform_Data_With_Spark manual__2025-06-27T07:14:33.133832+00:00 [running]> on host b867fdf2b33e
[2025-06-27T07:20:38.178+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spark_processing_pipeline' AIRFLOW_CTX_TASK_ID='Transform_Data_With_Spark' AIRFLOW_CTX_EXECUTION_DATE='2025-06-27T07:14:33.133832+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-27T07:14:33.133832+00:00'
[2025-06-27T07:20:38.182+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-27T07:20:38.254+0000] {base.py:84} INFO - Using connection ID 'spark_default' for task execution.
[2025-06-27T07:20:38.256+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: /home/***/.local/lib/python3.10/site-packages/pyspark/bin/spark-submit --master yarn --conf spark.master=yarn --conf spark.submit.deployMode=client --conf spark.yarn.appMasterEnv.YARN_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.yarn.appMasterEnv.HADOOP_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.executorEnv.YARN_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.executorEnv.HADOOP_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.yarn.queue=root.default --name arrow-spark --verbose --queue root.default /opt/***/include/scripts/transform_data.py /opt/***/data/foundation /opt/***/data/trusted
[2025-06-27T07:20:39.972+0000] {spark_submit.py:571} INFO - Using properties file: null
[2025-06-27T07:20:40.146+0000] {spark_submit.py:571} INFO - Parsed arguments:
[2025-06-27T07:20:40.147+0000] {spark_submit.py:571} INFO - master                  yarn
[2025-06-27T07:20:40.148+0000] {spark_submit.py:571} INFO - remote                  null
[2025-06-27T07:20:40.149+0000] {spark_submit.py:571} INFO - deployMode              client
[2025-06-27T07:20:40.150+0000] {spark_submit.py:571} INFO - executorMemory          null
[2025-06-27T07:20:40.151+0000] {spark_submit.py:571} INFO - executorCores           null
[2025-06-27T07:20:40.151+0000] {spark_submit.py:571} INFO - totalExecutorCores      null
[2025-06-27T07:20:40.152+0000] {spark_submit.py:571} INFO - propertiesFile          null
[2025-06-27T07:20:40.153+0000] {spark_submit.py:571} INFO - driverMemory            null
[2025-06-27T07:20:40.154+0000] {spark_submit.py:571} INFO - driverCores             null
[2025-06-27T07:20:40.155+0000] {spark_submit.py:571} INFO - driverExtraClassPath    null
[2025-06-27T07:20:40.156+0000] {spark_submit.py:571} INFO - driverExtraLibraryPath  null
[2025-06-27T07:20:40.157+0000] {spark_submit.py:571} INFO - driverExtraJavaOptions  null
[2025-06-27T07:20:40.157+0000] {spark_submit.py:571} INFO - supervise               false
[2025-06-27T07:20:40.158+0000] {spark_submit.py:571} INFO - queue                   root.default
[2025-06-27T07:20:40.159+0000] {spark_submit.py:571} INFO - numExecutors            null
[2025-06-27T07:20:40.160+0000] {spark_submit.py:571} INFO - files                   null
[2025-06-27T07:20:40.161+0000] {spark_submit.py:571} INFO - pyFiles                 null
[2025-06-27T07:20:40.161+0000] {spark_submit.py:571} INFO - archives                null
[2025-06-27T07:20:40.162+0000] {spark_submit.py:571} INFO - mainClass               null
[2025-06-27T07:20:40.163+0000] {spark_submit.py:571} INFO - primaryResource         file:/opt/***/include/scripts/transform_data.py
[2025-06-27T07:20:40.163+0000] {spark_submit.py:571} INFO - name                    arrow-spark
[2025-06-27T07:20:40.164+0000] {spark_submit.py:571} INFO - childArgs               [/opt/***/data/foundation /opt/***/data/trusted]
[2025-06-27T07:20:40.165+0000] {spark_submit.py:571} INFO - jars                    null
[2025-06-27T07:20:40.166+0000] {spark_submit.py:571} INFO - packages                null
[2025-06-27T07:20:40.167+0000] {spark_submit.py:571} INFO - packagesExclusions      null
[2025-06-27T07:20:40.167+0000] {spark_submit.py:571} INFO - repositories            null
[2025-06-27T07:20:40.168+0000] {spark_submit.py:571} INFO - verbose                 true
[2025-06-27T07:20:40.169+0000] {spark_submit.py:571} INFO - 
[2025-06-27T07:20:40.169+0000] {spark_submit.py:571} INFO - Spark properties used, including those specified through
[2025-06-27T07:20:40.170+0000] {spark_submit.py:571} INFO - --conf and those from the properties file null:
[2025-06-27T07:20:40.171+0000] {spark_submit.py:571} INFO - (spark.executorEnv.HADOOP_CONF_DIR,/opt/hadoop/etc/hadoop)
[2025-06-27T07:20:40.172+0000] {spark_submit.py:571} INFO - (spark.executorEnv.YARN_CONF_DIR,/opt/hadoop/etc/hadoop)
[2025-06-27T07:20:40.173+0000] {spark_submit.py:571} INFO - (spark.master,yarn)
[2025-06-27T07:20:40.173+0000] {spark_submit.py:571} INFO - (spark.submit.deployMode,client)
[2025-06-27T07:20:40.174+0000] {spark_submit.py:571} INFO - (spark.yarn.appMasterEnv.HADOOP_CONF_DIR,/opt/hadoop/etc/hadoop)
[2025-06-27T07:20:40.175+0000] {spark_submit.py:571} INFO - (spark.yarn.appMasterEnv.YARN_CONF_DIR,/opt/hadoop/etc/hadoop)
[2025-06-27T07:20:40.176+0000] {spark_submit.py:571} INFO - (spark.yarn.queue,root.default)
[2025-06-27T07:20:40.177+0000] {spark_submit.py:571} INFO - 
[2025-06-27T07:20:40.177+0000] {spark_submit.py:571} INFO - 
[2025-06-27T07:20:40.494+0000] {spark_submit.py:571} INFO - Main class:
[2025-06-27T07:20:40.494+0000] {spark_submit.py:571} INFO - org.apache.spark.deploy.PythonRunner
[2025-06-27T07:20:40.495+0000] {spark_submit.py:571} INFO - Arguments:
[2025-06-27T07:20:40.495+0000] {spark_submit.py:571} INFO - file:/opt/***/include/scripts/transform_data.py
[2025-06-27T07:20:40.496+0000] {spark_submit.py:571} INFO - null
[2025-06-27T07:20:40.497+0000] {spark_submit.py:571} INFO - /opt/***/data/foundation
[2025-06-27T07:20:40.497+0000] {spark_submit.py:571} INFO - /opt/***/data/trusted
[2025-06-27T07:20:40.498+0000] {spark_submit.py:571} INFO - Spark config:
[2025-06-27T07:20:40.499+0000] {spark_submit.py:571} INFO - (spark.app.name,arrow-spark)
[2025-06-27T07:20:40.499+0000] {spark_submit.py:571} INFO - (spark.app.submitTime,1751008840470)
[2025-06-27T07:20:40.500+0000] {spark_submit.py:571} INFO - (spark.executorEnv.HADOOP_CONF_DIR,/opt/hadoop/etc/hadoop)
[2025-06-27T07:20:40.501+0000] {spark_submit.py:571} INFO - (spark.executorEnv.YARN_CONF_DIR,/opt/hadoop/etc/hadoop)
[2025-06-27T07:20:40.502+0000] {spark_submit.py:571} INFO - (spark.master,yarn)
[2025-06-27T07:20:40.503+0000] {spark_submit.py:571} INFO - (spark.submit.deployMode,client)
[2025-06-27T07:20:40.504+0000] {spark_submit.py:571} INFO - (spark.submit.pyFiles,)
[2025-06-27T07:20:40.504+0000] {spark_submit.py:571} INFO - (spark.yarn.appMasterEnv.HADOOP_CONF_DIR,/opt/hadoop/etc/hadoop)
[2025-06-27T07:20:40.505+0000] {spark_submit.py:571} INFO - (spark.yarn.appMasterEnv.YARN_CONF_DIR,/opt/hadoop/etc/hadoop)
[2025-06-27T07:20:40.505+0000] {spark_submit.py:571} INFO - (spark.yarn.isPython,true)
[2025-06-27T07:20:40.506+0000] {spark_submit.py:571} INFO - (spark.yarn.queue,root.default)
[2025-06-27T07:20:40.507+0000] {spark_submit.py:571} INFO - Classpath elements:
[2025-06-27T07:20:40.507+0000] {spark_submit.py:571} INFO - 
[2025-06-27T07:20:40.508+0000] {spark_submit.py:571} INFO - 
[2025-06-27T07:20:40.509+0000] {spark_submit.py:571} INFO - 
[2025-06-27T07:20:41.619+0000] {spark_submit.py:571} INFO - üöÄ Starting data transformation...
[2025-06-27T07:20:41.620+0000] {spark_submit.py:571} INFO - üìç Input path: /opt/***/data/foundation
[2025-06-27T07:20:41.621+0000] {spark_submit.py:571} INFO - üìç Output path: /opt/***/data/trusted
[2025-06-27T07:20:41.760+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 INFO SparkContext: Running Spark version 3.5.1
[2025-06-27T07:20:41.763+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 INFO SparkContext: OS info Linux, 6.6.87.2-microsoft-standard-WSL2, amd64
[2025-06-27T07:20:41.765+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 INFO SparkContext: Java version 17.0.15
[2025-06-27T07:20:41.842+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-06-27T07:20:41.965+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 INFO ResourceUtils: ==============================================================
[2025-06-27T07:20:41.966+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-06-27T07:20:41.967+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 INFO ResourceUtils: ==============================================================
[2025-06-27T07:20:41.968+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 INFO SparkContext: Submitted application: TransformData
[2025-06-27T07:20:42.000+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:41 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-06-27T07:20:42.017+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO ResourceProfile: Limiting resource is cpus at 1 tasks per executor
[2025-06-27T07:20:42.022+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-06-27T07:20:42.101+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SecurityManager: Changing view acls to: ***
[2025-06-27T07:20:42.102+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SecurityManager: Changing modify acls to: ***
[2025-06-27T07:20:42.103+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SecurityManager: Changing view acls groups to:
[2025-06-27T07:20:42.104+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SecurityManager: Changing modify acls groups to:
[2025-06-27T07:20:42.105+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2025-06-27T07:20:42.441+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO Utils: Successfully started service 'sparkDriver' on port 38249.
[2025-06-27T07:20:42.476+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SparkEnv: Registering MapOutputTracker
[2025-06-27T07:20:42.522+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SparkEnv: Registering BlockManagerMaster
[2025-06-27T07:20:42.541+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-06-27T07:20:42.543+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-06-27T07:20:42.606+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-06-27T07:20:42.634+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-7748b943-1739-4012-9d2f-b4d38b7ff829
[2025-06-27T07:20:42.650+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-06-27T07:20:42.716+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-06-27T07:20:42.854+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2025-06-27T07:20:42.925+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:42 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2025-06-27T07:20:43.196+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 INFO DefaultNoHARMFailoverProxyProvider: Connecting to ResourceManager at resourcemanager/172.18.0.3:8032
[2025-06-27T07:20:43.777+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 INFO Configuration: resource-types.xml not found
[2025-06-27T07:20:43.778+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 INFO ResourceUtils: Unable to find 'resource-types.xml'.
[2025-06-27T07:20:43.789+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 INFO Client: Verifying our application has not requested more than the maximum memory capability of the cluster (8192 MB per container)
[2025-06-27T07:20:43.790+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 INFO Client: Will allocate AM container, with 896 MB memory including 384 MB overhead
[2025-06-27T07:20:43.791+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 INFO Client: Setting up container launch context for our AM
[2025-06-27T07:20:43.793+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 INFO Client: Setting up the launch environment for our AM container
[2025-06-27T07:20:43.801+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 INFO Client: Preparing resources for our AM container
[2025-06-27T07:20:43.841+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:43 WARN Client: Neither spark.yarn.jars nor spark.yarn.archive is set, falling back to uploading libraries under SPARK_HOME.
[2025-06-27T07:20:47.068+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:47 INFO Client: Uploading resource file:/tmp/spark-f55bea0b-84ae-4438-8f84-956c3fb6881a/__spark_libs__6987209927947194191.zip -> hdfs://namenode:9000/user/***/.sparkStaging/application_1751006797025_0003/__spark_libs__6987209927947194191.zip
[2025-06-27T07:20:53.003+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO Client: Uploading resource file:/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/pyspark.zip -> hdfs://namenode:9000/user/***/.sparkStaging/application_1751006797025_0003/pyspark.zip
[2025-06-27T07:20:53.064+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO Client: Uploading resource file:/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip -> hdfs://namenode:9000/user/***/.sparkStaging/application_1751006797025_0003/py4j-0.10.9.7-src.zip
[2025-06-27T07:20:53.284+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO Client: Uploading resource file:/tmp/spark-f55bea0b-84ae-4438-8f84-956c3fb6881a/__spark_conf__6200419371255167248.zip -> hdfs://namenode:9000/user/***/.sparkStaging/application_1751006797025_0003/__spark_conf__.zip
[2025-06-27T07:20:53.336+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO SecurityManager: Changing view acls to: ***
[2025-06-27T07:20:53.337+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO SecurityManager: Changing modify acls to: ***
[2025-06-27T07:20:53.338+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO SecurityManager: Changing view acls groups to:
[2025-06-27T07:20:53.339+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO SecurityManager: Changing modify acls groups to:
[2025-06-27T07:20:53.340+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2025-06-27T07:20:53.375+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO Client: Submitting application application_1751006797025_0003 to ResourceManager
[2025-06-27T07:20:53.655+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO Client: Deleted staging directory hdfs://namenode:9000/user/***/.sparkStaging/application_1751006797025_0003
[2025-06-27T07:20:53.658+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 ERROR SparkContext: Error initializing SparkContext.
[2025-06-27T07:20:53.659+0000] {spark_submit.py:571} INFO - org.apache.hadoop.yarn.exceptions.YarnException: Failed to submit application_1751006797025_0003 to YARN : Application application_1751006797025_0003 submitted by user *** to unknown queue: root.default
[2025-06-27T07:20:53.660+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.submitApplication(YarnClientImpl.java:336)
[2025-06-27T07:20:53.661+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.yarn.Client.submitApplication(Client.scala:220)
[2025-06-27T07:20:53.662+0000] {spark_submit.py:571} INFO - at org.apache.spark.scheduler.cluster.YarnClientSchedulerBackend.start(YarnClientSchedulerBackend.scala:62)
[2025-06-27T07:20:53.662+0000] {spark_submit.py:571} INFO - at org.apache.spark.scheduler.TaskSchedulerImpl.start(TaskSchedulerImpl.scala:235)
[2025-06-27T07:20:53.663+0000] {spark_submit.py:571} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:604)
[2025-06-27T07:20:53.664+0000] {spark_submit.py:571} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2025-06-27T07:20:53.665+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2025-06-27T07:20:53.666+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2025-06-27T07:20:53.666+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2025-06-27T07:20:53.668+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2025-06-27T07:20:53.669+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2025-06-27T07:20:53.670+0000] {spark_submit.py:571} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2025-06-27T07:20:53.671+0000] {spark_submit.py:571} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2025-06-27T07:20:53.671+0000] {spark_submit.py:571} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2025-06-27T07:20:53.672+0000] {spark_submit.py:571} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2025-06-27T07:20:53.673+0000] {spark_submit.py:571} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2025-06-27T07:20:53.674+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-06-27T07:20:53.675+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-06-27T07:20:53.676+0000] {spark_submit.py:571} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2025-06-27T07:20:53.677+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2025-06-27T07:20:53.677+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO SparkUI: Stopped Spark web UI at http://b867fdf2b33e:4040
[2025-06-27T07:20:53.683+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 WARN YarnSchedulerBackend$YarnSchedulerEndpoint: Attempted to send shutdown message before the AM has registered!
[2025-06-27T07:20:53.689+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 WARN YarnSchedulerBackend$YarnSchedulerEndpoint: Attempted to request executors before the AM has registered!
[2025-06-27T07:20:53.693+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO YarnClientSchedulerBackend: Shutting down all executors
[2025-06-27T07:20:53.696+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO YarnSchedulerBackend$YarnDriverEndpoint: Asking each executor to shut down
[2025-06-27T07:20:53.700+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO YarnClientSchedulerBackend: YARN client scheduler backend Stopped
[2025-06-27T07:20:53.725+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2025-06-27T07:20:53.740+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO MemoryStore: MemoryStore cleared
[2025-06-27T07:20:53.741+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO BlockManager: BlockManager stopped
[2025-06-27T07:20:53.749+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO BlockManagerMaster: BlockManagerMaster stopped
[2025-06-27T07:20:53.750+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 WARN MetricsSystem: Stopping a MetricsSystem that is not running
[2025-06-27T07:20:53.752+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2025-06-27T07:20:53.769+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO SparkContext: Successfully stopped SparkContext
[2025-06-27T07:20:53.770+0000] {spark_submit.py:571} INFO - Traceback (most recent call last):
[2025-06-27T07:20:53.771+0000] {spark_submit.py:571} INFO - File "/opt/***/include/scripts/transform_data.py", line 170, in <module>
[2025-06-27T07:20:53.773+0000] {spark_submit.py:571} INFO - transform_data(input_path, output_path)
[2025-06-27T07:20:53.774+0000] {spark_submit.py:571} INFO - File "/opt/***/include/scripts/transform_data.py", line 90, in transform_data
[2025-06-27T07:20:53.776+0000] {spark_submit.py:571} INFO - .getOrCreate()
[2025-06-27T07:20:53.777+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 497, in getOrCreate
[2025-06-27T07:20:53.777+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 515, in getOrCreate
[2025-06-27T07:20:53.778+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 203, in __init__
[2025-06-27T07:20:53.780+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 296, in _do_init
[2025-06-27T07:20:53.781+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 421, in _initialize_context
[2025-06-27T07:20:53.782+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1587, in __call__
[2025-06-27T07:20:53.783+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.10/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2025-06-27T07:20:53.785+0000] {spark_submit.py:571} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2025-06-27T07:20:53.787+0000] {spark_submit.py:571} INFO - : org.apache.hadoop.yarn.exceptions.YarnException: Failed to submit application_1751006797025_0003 to YARN : Application application_1751006797025_0003 submitted by user *** to unknown queue: root.default
[2025-06-27T07:20:53.788+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.submitApplication(YarnClientImpl.java:336)
[2025-06-27T07:20:53.788+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.yarn.Client.submitApplication(Client.scala:220)
[2025-06-27T07:20:53.789+0000] {spark_submit.py:571} INFO - at org.apache.spark.scheduler.cluster.YarnClientSchedulerBackend.start(YarnClientSchedulerBackend.scala:62)
[2025-06-27T07:20:53.789+0000] {spark_submit.py:571} INFO - at org.apache.spark.scheduler.TaskSchedulerImpl.start(TaskSchedulerImpl.scala:235)
[2025-06-27T07:20:53.790+0000] {spark_submit.py:571} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:604)
[2025-06-27T07:20:53.790+0000] {spark_submit.py:571} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2025-06-27T07:20:53.791+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2025-06-27T07:20:53.791+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2025-06-27T07:20:53.792+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2025-06-27T07:20:53.792+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2025-06-27T07:20:53.792+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2025-06-27T07:20:53.793+0000] {spark_submit.py:571} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2025-06-27T07:20:53.794+0000] {spark_submit.py:571} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2025-06-27T07:20:53.794+0000] {spark_submit.py:571} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2025-06-27T07:20:53.795+0000] {spark_submit.py:571} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2025-06-27T07:20:53.796+0000] {spark_submit.py:571} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2025-06-27T07:20:53.796+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-06-27T07:20:53.797+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-06-27T07:20:53.798+0000] {spark_submit.py:571} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2025-06-27T07:20:53.798+0000] {spark_submit.py:571} INFO - 
[2025-06-27T07:20:53.858+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO ShutdownHookManager: Shutdown hook called
[2025-06-27T07:20:53.859+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO ShutdownHookManager: Deleting directory /tmp/spark-f55bea0b-84ae-4438-8f84-956c3fb6881a
[2025-06-27T07:20:53.868+0000] {spark_submit.py:571} INFO - 25/06/27 07:20:53 INFO ShutdownHookManager: Deleting directory /tmp/spark-731c004e-e397-43a7-91e5-11e6ad17b613
[2025-06-27T07:20:53.974+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-27T07:20:54.015+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: /home/***/.local/lib/python3.10/site-packages/pyspark/bin/spark-submit --master yarn --conf spark.master=yarn --conf spark.submit.deployMode=client --conf spark.yarn.appMasterEnv.YARN_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.yarn.appMasterEnv.HADOOP_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.executorEnv.YARN_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.executorEnv.HADOOP_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.yarn.queue=root.default --name arrow-spark --verbose --queue root.default /opt/***/include/scripts/transform_data.py /opt/***/data/foundation /opt/***/data/trusted. Error code is: 1.
[2025-06-27T07:20:54.021+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=spark_processing_pipeline, task_id=Transform_Data_With_Spark, run_id=manual__2025-06-27T07:14:33.133832+00:00, execution_date=20250627T071433, start_date=20250627T072037, end_date=20250627T072054
[2025-06-27T07:20:54.046+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 17 for task Transform_Data_With_Spark (Cannot execute: /home/***/.local/lib/python3.10/site-packages/pyspark/bin/spark-submit --master yarn --conf spark.master=yarn --conf spark.submit.deployMode=client --conf spark.yarn.appMasterEnv.YARN_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.yarn.appMasterEnv.HADOOP_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.executorEnv.YARN_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.executorEnv.HADOOP_CONF_DIR=/opt/hadoop/etc/hadoop --conf spark.yarn.queue=root.default --name arrow-spark --verbose --queue root.default /opt/***/include/scripts/transform_data.py /opt/***/data/foundation /opt/***/data/trusted. Error code is: 1.; 670)
[2025-06-27T07:20:54.072+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2025-06-27T07:20:54.127+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-27T07:20:54.142+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
